src/levo/cloudflare/har.ts

```diff
@@ -132,8 +132,25 @@ export const buildHarFromRequestResponse = async (
 	return har;
 };
 
-// Only report request/response pairs when response content type is application/json
+
+const supportedMimeTypes = [
+	"application/json",
+	"application/ld+json",
+	"text/json",
+	"application/x-www-form-urlencoded",
+	"application/grpc",
+	"multipart/form-data",
+	"application/xml",
+	"text/xml",
+	"text/plain",
+]
+// Only report request/response pairs for supported response MIME types
 export const shouldSendToLevo = (request: Request, response: Response) => {
-	const contentType = response.headers.get("Content-Type") || "";
-	return contentType.toLowerCase().includes("application/json");
+	const contentType = response.headers.get("Content-Type")?.toLowerCase();
+	if (!contentType) {
```

Comment:
@ricekot Could this be too verbose for the customer environment? IMHO this warning won't add too much value to the customer logs and we'll know the Content-Type is missing by looking into the traces on our side, right?